lib_test/ounit/dune

```diff
@@ -0,0 +1,9 @@
+(executable
+  (name       test)
+  (libraries  dns ounit pcap-format)
+  (preprocess (pps ppx_cstruct)))
+
+(alias
+  (name    runtest)
+  (deps    test.exe (glob_files *.pcap) (glob_files *.zone))
+  (action  (run ${<})))
```

Comment:
In a dune file, that probably tries to run the wrong thing.

Reply:
ha yes, good catch